lib/browser.ex

```diff
@@ -467,7 +467,7 @@ defmodule Browser do
   end
 
   def ios_version(input) do
-    Enum.at(Regex.run(~r/OS (\d+)/, Ua.to_ua(input)) || [], 1)
+    Enum.at(Regex.run(~r/OS ([\d.]+)/, Ua.to_ua(input)) || [], 1)
```

Comment:
```suggestion
    Enum.at(Regex.run(~r/OS ([\d\.]+)/, Ua.to_ua(input)) || [], 1)
```

It's better to explicitly tell the regex to look for dots, not for anything.